zsyscall_windows.go

```diff
@@ -109,3 +111,15 @@ func ApplyControlToken(context *CtxtHandle, input *SecBufferDesc) (ret syscall.E
 	ret = syscall.Errno(r0)
 	return
 }
+
+func MakeSignature(context *CtxtHandle, qop uint32, message *SecBufferDesc, messageseqno uint32) (ret syscall.Errno) {
+	r0, _, _ := syscall.Syscall6(procMakeSignature.Addr(), 4, uintptr(unsafe.Pointer(context)), uintptr(qop), uintptr(unsafe.Pointer(message)), uintptr(messageseqno), 0, 0)
+	ret = syscall.Errno(r0)
+	return
+}
+
+func ValidateSignature(context *CtxtHandle, message *SecBufferDesc, messageseqno uint32, qop *uint32) (ret syscall.Errno) {
```

Comment:
This function is called ValidateSignature, but syscall.go says:

//sys	VerifySignature ...

How is that possible, if you run `go generate` command?

Reply:
I don't normally develop on Windows and although I can cross compile for Windows from a Linux or OS X host I couldn't work out how to run `go generate` without being on an actual Windows host so I'm afraid I just ran the `go generate` and copied the diff of `zsyscall_windows.go` across. The GSSAPI equivalent function is `gss_verify_mic()` so I've gotten my verify and validate mixed up, apologies.

When I ran this with go 1.9.x it made more changes to the file other than just add these new functions but I figured that would probably break compatibility with older golang's so I haven't included them.

Reply:
> I don't normally develop on Windows and although I can cross compile for Windows from a Linux or OS X host I couldn't work out how to run go generate without being on an actual Windows host

`go generate` only worked on Windows. But now I moved it to a file that is visible on any OS, so you should be able to use it too. I also added your functions here https://github.com/alexbrainman/sspi/commit/bc15a409b6322b891e172a20c0bb8f0924b0cda2